.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+*Replace this paragraph with a description of what this PR is changing or adding, and why. Consider including before/after screenshots.*
+
+*List which issues are fixed by this PR. You must list at least one issue.*
+
+## Pre-launch Checklist
+
+- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
+- [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
+- [ ] I read the [Flutter Style Guide] _recently_, and have followed its advice.
+- [ ] I signed the [CLA].
+- [ ] I listed at least one issue that this PR fixes in the description above.
+- [ ] I updated/added relevant documentation (doc comments with `///`).
+- [ ] I added new tests to check the change I am making, or there is a reason for not adding tests.
+- Existing and new test status: ![build.yaml badge]
+
+If you need help, consider asking for advice on the #hackers-devexp-📐 channel on [Discord].
```

Comment:
point users to #devtools

Reply:
Point to the flutter chat wiki instead. You can say, "consider asking for help on Discord"
see link here: https://github.com/flutter/devtools/pull/4926/files#diff-2c22fff01f45b14288a3152294267b837c6f428009d4d1e0c09d23c64ba6382eR118